x-pack/plugins/data_visualizer/public/application/common/components/stats_table/components/field_data_expanded_row/choropleth_map.tsx

```diff
@@ -104,14 +105,19 @@ export const ChoroplethMap: FC<Props> = ({ stats, suggestion }) => {
   );
 
   return (
-    <EuiFlexItem data-test-subj={'fileDataVisualizerChoroplethMapTopValues'}>
-      <div style={{ width: '100%', minHeight: 300 }}>
+    <ExpandedRowPanel
+      dataTestSubj={'fileDataVisualizerChoroplethMapTopValues'}
+      className={'dataVisualizerPanelWrapper '}
```

Comment:
Nit - is the space needed after the className here?

Reply:
Fixed here https://github.com/elastic/kibana/pull/112870/commits/6c0c8505076c4152d0ddacfb34b3909598c08403